### PR TITLE
Wening august changes

### PR DESCRIPTION
**Pull request by BMWENING**
Send email to user and staff when removed from visiting facility (both manual and automatic when removed from home)
Disable transferring to own facility (e.g. ZDV->ZDV)
Corrected transfer rejected email to include proper usa2 email
Created staff appointment transfer flag for both facility and ZHQ staff appointments

### CODE REVIEW

File: app/Classes/RoleHelper.php [hunk 1]
```diff
@@ -580,7 +580,7 @@ public static function getStaff($facility = null, $getVATUSA = true)
         return $staff;
     }
 
-    public static function addFacilityStaffPosition($facility, $cid, $pos) {
+    public static function addFacilityStaffPosition($facility, $cid, $pos, $xfer = "true") {
```
Comment (Jonhasacat) [inline app/Classes/RoleHelper.php:583]: $xfer should be a boolean, not a string

File: app/Http/Controllers/MyController.php [hunk 2]
```diff
@@ -120,7 +120,7 @@ public function postSelect(Request $request) {
         }
 
         if ($_POST['facility'] == "0") {
-            return redirect('/my/transfer')->with('error', "You didn't select a facility!");
+            return redirect('/my/select')->with('error', "You didn't select a facility!");
```
Comment (Jonhasacat) [inline app/Http/Controllers/MyController.php:123]: Why did you change this?

**Review by Jonhasacat — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/MyController.php (hunk 2 above)
Comment (BMWENING) [reply app/Http/Controllers/MyController.php:124]: The user didn't start on the transfer page, they started on the select page. So I corrected it to return back to the page they were originally on.

**Review by Jonhasacat — APPROVED** [review]
